.pre-commit-config.yaml

```diff
@@ -12,7 +12,10 @@ repos:
   rev: 'v1.10.0'
   hooks:
     - id: mypy
-      additional_dependencies: ['polars==0.20.10', 'pytest==8.0.1']
+      additional_dependencies:
+        - "polars==0.20.10"
+        - "pytest==8.0.1"
+        - "typing-extensions==4.12.2"
```

Comment:
isn't this already a dependency of mypy?

https://github.com/python/mypy/blob/d1d3c780c7a2d30b2a038903289ea7487303a218/pyproject.toml#L1-L15

Reply:
It is but, since we are explicitly using it in some imports, I believe it is better to have pinned it.

Just because *explicit is better than implicit*